protocol/v2/qbft/instance/prepare.go

```diff
@@ -26,6 +26,12 @@ func (i *Instance) uponPrepare(logger *zap.Logger, msg *specqbft.ProcessingMessa
 		return nil // uponPrepare was already called
 	}
 
+	proposedRoot := i.State.ProposalAcceptedForCurrentRound.QBFTMessage.Root
+	logger.Debug("📬 got prepare message",
+		fields.Round(i.State.Round),
+		zap.Any("prepare-signers", i.State.ProposalAcceptedForCurrentRound.SignedMessage.OperatorIDs),
+		fields.Root(proposedRoot))
```

Comment:
maybe we can add quorum before as a field to this log?

Reply:
@y0sher there's a separate log for quorum below